wolfcrypt/src/evp.c

```diff
@@ -1366,16 +1366,27 @@ int wolfSSL_EVP_add_digest(const WOLFSSL_EVP_MD *digest)
  *
  * return WOLFSSL_SUCCESS on success
  */
+#ifdef WOLFSSL_QT
```

Comment:
Should switch on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT, as well as function body.

Reply:
Guard by OPNESSL_VERSION_NUMBER instead of WOLFSSL_QT